### ISSUE TITLE
Extract the CSV writer to it's own Ruby class

### ISSUE THREAD

**Issue by rietta** [rietta/git_time_extractor#12]

**Comment by benzkji** [comment]
...and let me choose if I want JSON, just in case... ;-)

**Comment by rietta** [comment]
Right. That's the benefit of extracting the output into a writer pattern. Then any number of writers can be implemented as subclasses/implementers of the interface.
